src/simulation/elements/FIRE.cpp

```diff
@@ -88,9 +88,10 @@ int Element_FIRE_update(UPDATE_FUNC_ARGS)
 		}
 		break;
 	case PT_LAVA:
+
+		float pres = sim->pv[y / CELL][x / CELL];
```

Comment:
This doesn't compile. Can you add some brackets around the PT_LAVA case to fix it?

Reply:
Did as you said but it also had no brackets before so i am not sure if it's fixed now.